app/src/main/java/com/example/parkingsystemkotlin/activity/ParkingSpaceReservationActivity.kt

```diff
@@ -38,6 +38,7 @@ class ParkingSpaceReservationActivity : AppCompatActivity(), DatePickerDialog.On
     private fun setListeners() {
         binding.buttonParkingSpaceReservationPickerBegin.setOnClickListener { presenter.onButtonParkingSpaceReservationPickerPressed(this) }
         binding.buttonParkingSpaceReservationPickerEnd.setOnClickListener { presenter.onButtonParkingSpaceReservationPickerPressed(this) }
+        binding.buttonParkingSpaceReservationSave.setOnClickListener { presenter.onButtonParkingSpaceReservationSavePressed() }
```

Comment:
In this case you could use with(binding){ ... }

Reply:
Done!